lib/SILOptimizer/Mandatory/TFDeabstraction.cpp

```diff
@@ -1885,6 +1913,38 @@ void TFDeabstraction::checkAndCanonicalizeAttributes() {
                    diag::tf_op_misuse, error)
             .highlight(loc.getSourceRange());
           isError = true;
+
+          // If we have more specific information about what went wrong, emit a
```

Comment:
Is this a TODO?

Reply:
No, this is describing what the code immediately below it does.  I deleted the extraneous blank line.